### ISSUE TITLE
enable the registry-mirror flag in vela-docker

### ISSUE THREAD

**Issue by jbrockopp** [go-vela/community#38]
## Description

Original story: https://github.com/go-vela/vela-docker/issues/40

Add the ability to use the `registry-mirror` flag in kaniko

Flag documentation:

https://github.com/GoogleContainerTools/kaniko#--registry-mirror

## Value

Allow users to pull from a different registry mirrors

## Definition of Done

The `registry-mirror` flag is useable in the plugin.

## Effort  (Optional)

2 days

## Impacted Personas (Optional)

users of vela-docker

**Comment by JordanSussman** [comment]
This is likely more important with the latest data transfers updates on https://www.docker.com/pricing

![image](https://user-images.githubusercontent.com/8935518/90288962-82aefa00-de40-11ea-8f35-065932a0b369.png)


TL;DR docker is imposing limits on pulls for unauthenticated users starting in November.

**Comment by kneal** [comment]
This one should be good to go:
https://github.com/go-vela/vela-kaniko/pull/56

**Comment by kneal** [comment]
Docker is now kaniko
